### ISSUE TITLE
[Bug] Downgrade and multi-projection 

### ISSUE THREAD

**Issue by youdz** [angular/angular#7727]
### Bug Report

Angular version: 2.0.0-beta.11
Browser: [all]
Language: [Typescript]
### Description

When downgrading an Angular 2 component to use in an Angular 1 application, the downgraded directive crashes if the original component uses multi-projection with more than 1 slot, for instance:

``` html
<ng-content select=".foo"></ng-content>
<ng-content select=".bar"></ng-content>
```

The error thrown is `Error: The component {XXX} has {N} <ng-content> elements, but only 1 slots were provided.`
### Reproduce

See [this plunker](http://plnkr.co/edit/JvjE9fodGnauwQNO7j5v?p=preview) for a minimal example that reproduces the bug.

Thanks.

**Comment by Blackbaud-PatrickOFriel** [comment]
In the release candidate, when using multi-projection, everything seems to be getting placed in the first `ng-content` area, regardless of selector

**Comment by petebacondarwin** [comment]
Fixed by #12675
